packages/vue/index.js

```diff
@@ -136,5 +136,5 @@ export {
   SfStoreLocator,
   SfTable,
   SfTabs,
-  SfTopBar,
```

Comment:
we can leave it :)

Reply:
I think this caused by auto generator script :)